src/shared/services/bolt/boltMappings.js

```diff
@@ -219,3 +219,31 @@ export const retrieveFormattedUpdateStatistics = (result) => {
 export const flattenProperties = (rows) => {
   return rows.map((row) => row.map((entry) => (entry && entry.properties) ? entry.properties : entry))
 }
+
+export const applyGraphTypes = (item) => {
+  if (Array.isArray(item)) {
+    return item.map(arrayItem => applyGraphTypes(arrayItem))
```

Comment:
Minor thing, but it's easier to read if you do this `item.map(applyGraphTypes)`.